<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hani3K</class>
 <widget class="QMainWindow" name="hani3K">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>524</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>hani3K</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QTextBrowser" name="book_info">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Word book:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="word_book">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Use Default</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="load_btn">
                 <property name="text">
                  <string>Load Book</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>from</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>to</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="from_bsr"/>
               </item>
               <item>
                <widget class="QLineEdit" name="to_bsr"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="learn_type">
             <item>
              <property name="text">
               <string>All</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Star</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Unknown</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
          <item>
           <widget class="QTextBrowser" name="word_bsr"/>
          </item>
          <item>
           <widget class="QTextBrowser" name="meaning_bsr"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="next_btn">
              <property name="text">
               <string>Next(D)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="show_btn">
              <property name="text">
               <string>Show(S)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="star_btn">
              <property name="text">
               <string>Star(W)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="known_btn">
              <property name="text">
               <string>Known(A)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QTextBrowser" name="log_bsr"/>
         </item>
         <item>
          <widget class="QPushButton" name="save_log_btn">
           <property name="text">
            <string>Save log</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>load_btn</sender>
   <signal>clicked()</signal>
   <receiver>hani3K</receiver>
   <slot>load_book()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>hani3K</receiver>
   <slot>start_learn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_log_btn</sender>
   <signal>clicked()</signal>
   <receiver>hani3K</receiver>
   <slot>save_log()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next_btn</sender>
   <signal>clicked()</signal>
   <receiver>hani3K</receiver>
   <slot>next_word()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_btn</sender>
   <signal>clicked()</signal>
   <receiver>hani3K</receiver>
   <slot>show_meaning()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>star_btn</sender>
   <signal>clicked()</signal>
   <receiver>hani3K</receiver>
   <slot>star_word()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>known_btn</sender>
   <signal>clicked()</signal>
   <receiver>hani3K</receiver>
   <slot>set_known_word()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>hani3K</receiver>
   <slot>use_default()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>load_book()</slot>
  <slot>start_learn()</slot>
  <slot>save_log()</slot>
  <slot>next_word()</slot>
  <slot>show_meaning()</slot>
  <slot>star_word()</slot>
  <slot>set_known_word()</slot>
  <slot>use_default()</slot>
 </slots>
</ui>
